editor/script_create_dialog.cpp

```diff
@@ -168,9 +161,9 @@ void ScriptCreateDialog::_load_exist() {
 	String path = file_path->get_text();
 	RES p_script = ResourceLoader::load(path, "Script");
 	if (p_script.is_null()) {
-		alert->get_ok()->set_text(TTR("Ugh"));
+		alert->get_ok()->set_text(TTR("OK"));
 		alert->set_text(vformat(TTR("Error loading script from %s"), path));
-		alert->popup_centered_minsize();
+		alert->popup_centered();
```

Comment:
Meh it collapses :D 
Oki, now I understand how it works, will fix all places it breaks :) Thx for check!